app/components/post_body/index.js

```diff
@@ -26,16 +41,45 @@ function mapStateToProps(state, ownProps) {
         isPending = false;
     }
 
+    const isEphemeralPost = isPostEphemeral(post);
+    const isSystemPost = isSystemMessage(post);
+
+    const channel = getCurrentChannel(state);
+    const user = getCurrentUser(state);
+    const teamMember = getCurrentTeamMembership(state);
+    const channelMember = getMyCurrentChannelMembership(state);
+    const {config, license} = state.entities.general;
+
+    const isUserCanManageMembers = canManageMembers(channel, user, teamMember, channelMember, config, license);
+
+    let isPostAddChannelMember = false;
+    if (
+        (channel.type === General.PRIVATE_CHANNEL || channel.type === General.OPEN_CHANNEL) &&
+        isUserCanManageMembers &&
+        isEphemeralPost &&
+        post.props &&
+        post.props.add_channel_member
+    ) {
+        isPostAddChannelMember = true;
+    }
+
+    let addMemberProps = {};
+    if (post.props && post.props.add_channel_member) {
+        addMemberProps = post.props.add_channel_member;
```

Comment:
We are already passing the post props, do we really need this here?

Reply:
You're right, this is no longer needed.